### PR TITLE
Replace travis/appveyor with GHA; use workflow for deploying pkgdown

### PR DESCRIPTION
**Pull request by cpsievert**

### CODE REVIEW

**Comment by schloerke** [comment]
- [ ] Remove appveyor file

File: README.md [hunk 1]
```diff
@@ -43,43 +42,35 @@ less complex and more composible.
...
+
+    library(sass)
+    sass(input = "
+      $size: 50%;
+      foo { margin: $size * .33; }
+    ")
+
+    foo {
+      margin: 16.5%;
+    }
```
Comment (cpsievert) [inline README.md:73]: Whoa, is this due to a change in rmarkdown?

Comment (schloerke) [reply README.md:73]: Yes. Seems like it. Or pandoc?

Comment (schloerke) [reply README.md:73]: Oh no. Not good

Comment (cpsievert) [reply README.md]: I just manually fixed it for now (I think it's some (hopefully) temporary pandoc weirdness)

**Pull request merged**